base_layer/core/src/types.rs

```diff
@@ -67,6 +68,9 @@ pub type MessageHash = Vec<u8>;
 /// Specify the range proof type
 pub type RangeProofService = DalekRangeProofService;
 
+/// Specify the Proof of Work
+pub type POW = MockProofOfWork;
```

Comment:
Much prefer the original name (ProofOfWork) and you can use namespacing to disambiguate the two

Reply:
I dislike using that as it can become confusing if the same object/type is named differently in different files. I have changed the trait name as this is not widely used currently. I just used the trait to give an indication as to what function's we need to implement in the actual PoW although the code isnt technically using the interface.